shared/src/commonMain/kotlin/com/bunbeauty/shared/data/repository/SuggestionRepository.kt

```diff
@@ -0,0 +1,36 @@
+package com.bunbeauty.shared.data.repository
+
+import com.bunbeauty.shared.data.network.api.NetworkConnector
+import com.bunbeauty.shared.domain.model.Suggestion
+import com.bunbeauty.shared.domain.repo.SuggestionRepo
+import com.bunbeauty.shared.extension.dataOrNull
+
+class SuggestionRepository(
+    private val networkConnector: NetworkConnector,
+) : SuggestionRepo {
+
+    private val cache: MutableMap<String, List<Suggestion>> = mutableMapOf()
+
+    override suspend fun getSuggestionList(token: String, query: String, cityUuid: String): List<Suggestion>? {
```

Comment:
nullable list? 
can we use empty list for case with null?

Reply:
empty list is a valid response - everything works fine, but nothing was found
null means something went wrong